client/tests/e2e/snapshots/utils.js

```diff
@@ -35,14 +35,17 @@ async function cleanAndNormalizeSnapshot(page) {
         const text = el.textContent.trim();
         if (
           text.includes('Ready for Review') ||
-          text.includes('Review in Progress')
+          text.includes('Review in Progress') ||
+          (text.includes('Days') && text.includes('Past'))
```

Comment:
You'll also want to check for ['Days Away'](https://github.com/w3c/aria-at-app/blob/7f6d795306253035bd65aa574721f02a502bf80d/client/components/DataManagement/DataManagementRow/index.jsx#L940)